examples/BasicSerialLogger/BasicSerialLogger.ino

```diff
@@ -19,30 +25,23 @@ const char globalProgmemMessageWithRamString[] PROGMEM
 
 // Need to remember that backslash must be escaped in the string text
 // Banner generated with https://devops.datenkollektiv.de/banner.txt/index.html
-const char banner0[] PROGMEM = " ___   ___   ___   _ _    _                 ";
-const char banner1[] PROGMEM = "| _ \\ | __| |_  ) | | |  | |     ___   __ _ ";
-const char banner2[] PROGMEM = "|   / | _|   / /  |_  _| | |__  / _ \\ / _` |";
-const char banner3[] PROGMEM = "|_|_\\ |_|   /___|   |_|  |____| \\___/ \\__, |";
-const char banner4[] PROGMEM = "                                      |___/ ";
+const PROGMEM char banner[5][]  = { "   ___    ____   ___   ____   __             ",
+                                    "  / _ \\  / __/  |_  | / / /  / /  ___   ___ _",
+                                    " / , _/ / _/   / __/ /_  _/ / /__/ _ \\ / _ `/",
+                                    "/_/|_| /_/    /____/  /_/  /____/\\___/ \\_, / ",
+                                    "                                      /___/  "}
 
 void setup()
 {
    // configure serial port baudrate
-   Serial.begin(57600);
+   Serial.begin(115200);
    // set serial port appender
    rf24Logger.setAppender(&rf24SerialLogAppender);
 
    // display banner
-   rf24Logger.info((const __FlashStringHelper*) vendorID,
-         (const __FlashStringHelper*) banner0);
-   rf24Logger.info((const __FlashStringHelper*) vendorID,
-         (const __FlashStringHelper*) banner1);
-   rf24Logger.info((const __FlashStringHelper*) vendorID,
-         (const __FlashStringHelper*) banner2);
-   rf24Logger.info((const __FlashStringHelper*) vendorID,
-         (const __FlashStringHelper*) banner3);
-   rf24Logger.info((const __FlashStringHelper*) vendorID,
-         (const __FlashStringHelper*) banner4);
+   for (uint8_t i = 0; i < 5; ++i)
+      rf24Logger.info((const __FlashStringHelper*) vendorID, (const __FlashStringHelper*) banner[i]);

```

Comment:
I prefere to add here those ```{ }``` so it has a better readibility of what is being executed in the **for** loop - even if the loop has only one-line execution.

Reply:
Brackets also added.